### DISCUSSION TITLE
Possible return of balena etcher cask

### DISCUSSION THREAD

**Discussion by MTCoster** [Homebrew/discussions#3943] [Casks] [answered]
Since the cask for balena etcher was disabled in Homebrew/homebrew-cask#135413 due to the ridiculous release cadence, there have been no further releases.

My understanding of the issue is that they've hooked their CI up to automatically make a release when a dependency is updated, but that this was only recently fixed to work correctly resulting in the slew of releases over a short time period as it cleared the backlog.

Now the backlog appears to be cleared and no further releases have been made in the past ~two weeks~ 11 days, is it worth re-adding support for this app?

**Comment by SMillerDev** [comment]
Since upstream mentioned that that  cadence is by design I don't feel comfortable re-enabling the cask.

**Comment by p-linnane** [comment, marked as answer]
They seem to be having a number of issues over there right now. I [opened a PR](https://github.com/balena-io/etcher/pull/3875) 5 days ago to remove `brew` instructions in the README and they aren't able to figure out how to accept external contributions right now due to their CI config. This, in addition to the crazy release cadence being "paused" right now, does not give me a good feeling. Once they sort this stuff out I would review a PR to readd.

**Comment by MTCoster** [comment]
Update: things seem to be getting more sensible upstream:

> prerelease has been implemented in https://github.com/balena-io/etcher/pull/3868 and renovate (re)enabled in https://github.com/balena-io/renovate-config/pull/271.

([ref](https://github.com/balena-io/etcher/issues/3853#issuecomment-1338266521))

I'll give it a couple weeks to make sure the pre-releases are marked as expected, then open a PR to re-enable the cask (unless anyone has any objections).
